### ISSUE TITLE
foundry v10 AA 3.3.13: checkConcentration is throwing an error

### ISSUE THREAD

**Issue by tposney** [otigon/automated-jb2a-animations#480]
I was tinkering with midi-qol concentration and happened to notice that checkConcentration is throwing an error when doing canvas.background.tiles.

I think this version works (but not really sure how to create an AA with a tile)
```js
export async function checkConcentration(effect) {
    const aaDebug = game.settings.get("autoanimations", "debug")

    // Check effect label and return if it is not equal to "concentrating"
    const label = effect.label || "";
    if (label.toLowerCase() !== "concentrating") { return; }

    // Get Originating Item. If no Origin, return
    const origin = effect.origin
    if (!origin) {
        if (aaDebug) { aaDebugger("Failed to find an Origin for Concentration") }
        return;
    }

    // Get arrays of Background and Foreground Tiles with the A-A Origin flag UUID matching the Effect Origin
    const tileIdArray = canvas.tiles.placeables.filter(i => i.flags?.autoanimations?.origin === origin).map(tile => tile.id);
    if (tileIdArray.length < 1) {
        if (aaDebug) { aaDebugger("Failed to find any Tiles tied to Concentration") }
        return;
    }
    socketlibSocket.executeAsGM("removeTile", tileIdArray)
    //Sequencer.EffectManager.endEffects({ origin: origin })
}
```

**Comment by otigon** [comment]
Thanks for letting me know. It looks like I forgot to update that for the new Canvas. I'll get that smashed out today

**Comment by otigon** [comment]
Error is fixed as of version 3.3.14. It looks like tiles are all in a single layer now under ``canvas.tiles`` instead of separated by background/foreground.
